src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -169,7 +171,7 @@ export const useSwapper = () => {
     sellAsset: Asset
     buyAsset: Asset
     amount: string
-  }): Promise<Quote<ChainTypes> | undefined> => {
+  }): Promise<BuildQuoteTxOutput | undefined> => {
```

Comment:
```suggestion
  }): Promise<BuildQuoteTxOutput> => {
```

if this throws, we can safely tighten to remove the `| undefined` - please take a look and tighten if possible